clang-tools-extra/docs/clang-tidy/checks/bugprone/unsafe-functions.rst

```diff
@@ -86,6 +123,22 @@ Options
    this option enables.
    Default is `true`.
 
+.. option:: ReportDefaultFunctions
+
+    When `true`, the check reports the default set of functions.
+    Default is `true`.
+
+.. option:: CustomNormalFunctions
+
+    A comma-separated list of regular expressions, their replacements, and an
+    optional reason. For more information, see `Custom functions
+    <unsafe-functions.html#custom-functions>`_.
+
+.. option:: CustomAnnexKFunctions
+
+    A comma-separated list of regular expressions, their replacements, and an
+    optional reason. For more information, see `Custom functions
+    <unsafe-functions.html#custom-functions>`_.
```

Comment:
What do you think about:

> A semicolon-separated list of specifications, each containing a regular expression, a replacement, and an optional reason, separated by a comma.

? I think there is still room for improvement, but I'm worried about the not mentioned semicolon/structure (even though you link to the documentation).

Reply:
> A semicolon-separated list of custom functions to be matched. A matched
    function contains a regular expression, the name of the replacement
    function, and an optional reason, separated by comma.

Should be good enough for these purposes.